src/sections/dashboard/index.js

```diff
@@ -11,207 +14,170 @@ import {
-                        avatarTitle="Saray Pacheco" />
-
-                    <Message
-                        avatarSrc="assets/images/user3.jpg"
-                        userName="Reinier Guerra"
-                        date="7:25 am"
-                        content="A rainbow is a meteorological phenomenon that is caused by reflection, refraction and dispersion of light in water droplets resulting in a spectrum …"
-                        avatarTitle="Reinier Guerra" />
-
-                    <Message
-                        avatarSrc="assets/images/user1.jpg"
-                        userName="Jose Leandro Torres"
-                        date="yesterday"
-                        content="A rainbow is a meteorological phenomenon that is caused by reflection, refraction and dispersion of light in water droplets resulting in a spectrum …"
-                        avatarTitle="Jose Leandro Torres" />
-
-                    <Message
-                        avatarSrc="assets/images/user4.jpg"
-                        userName="Tahimi León"
-                        date="yesterday"
-                        content="A rainbow is a meteorological phenomenon that is caused by reflection, refraction and dispersion of light in water droplets resulting in a spectrum …"
-                        avatarTitle="Tahimi León" />
-
-                    <Message
-                        avatarIcon={<PersonIcon />}
-                        userName="Carlos Miguel"
-                        date="Monday"
-                        content="A rainbow is a meteorological phenomenon that is caused by reflection, refraction and dispersion of light in water droplets resulting in a spectrum …"
-                        avatarTitle="Carlos Miguel"
-                        hasDivider={false} />
-
-                    <div className="react-rainbow-admin-dashboard_card-message--divider" />
-
-                    <div className="rainbow-p-top_small rainbow-align-content_center">
-                        <a
-                            className="react-rainbow-admin-dashboard_card-message--footer-link rainbow-color_brand"
-                            href="javascript:void(0);"
-                            target="_blank"
-                            rel="noopener noreferrer">
+                        </h1>
+
+                        <LastMessagesList lastMessages={lastMessages} />
+
+                        <div className="react-rainbow-admin-dashboard_card-message--divider" />
+
+                        <div className="rainbow-p-top_small rainbow-align-content_center">
+                            <a
+                                className="react-rainbow-admin-dashboard_card-message--footer-link rainbow-color_brand"
+                                href="javascript:void(0);"
+                                target="_blank"
+                                rel="noopener noreferrer">
```

Comment:
put the card wrapper, divider and link inside LastMessageList, the same for NewUsersList